src/main/scala/com/microsoft/hyperspace/index/sources/delta/DeltaLakeRelation.scala

```diff
@@ -46,8 +46,8 @@ class DeltaLakeRelation(spark: SparkSession, override val plan: LogicalRelation)
   override def allFiles: Seq[FileStatus] = plan.relation match {
     case HadoopFsRelation(location: TahoeLogFileIndex, _, _, _, _, _) =>
       location
-        .getSnapshot(stalenessAcceptable = false)
-        .filesForScan(projection = Nil, location.partitionFilters, keepStats = false)
+        .getSnapshot
+        .filesForScan(projection = Nil, location.partitionFilters)
```

Comment:
@sezruby in Delta 8.0, there were some changes in the API. Could you confirm if these changes are fine?